spaceship/spec/tunes/fixtures/ratings_summary.json

```diff
@@ -1,94 +1,16 @@
 {
```

Comment:
Sorry, I missed this before, but should this be `ratings.json` rather than `ratings_summary.json`?